macros/base/start_print.cfg

```diff
@@ -349,9 +349,11 @@ gcode:
     {% if ercf_enabled %}
         {% if ercf_reset_stats_on_start_print %}
             ERCF_RESET_STATS
-        {% endif %}            
-        ERCF_CHANGE_TOOL STANDALONE=1 TOOL={INITIAL_TOOL}
-        ### _DO_ERROR_ERCF_PAUSE
+        {% endif %}
+        {% if printer.ercf.tool|int != INITIAL_TOOL or printer.ercf.filament != "Loaded" %}
+            ERCF_CHANGE_TOOL STANDALONE=1 TOOL={INITIAL_TOOL}
+        {% endif %}
+        _DO_ERROR_ERCF_PAUSE
```

Comment:
where did you readd this function?

Reply:
This function is a custom Klippain function added to be able to stop the START_PRINT routine in case there is an error on the ERCF